.github/workflows/icu_ci.yml

```diff
@@ -0,0 +1,80 @@
+# Copyright (C) 2016 and later: Unicode, Inc. and others.
+# License & terms of use: http://www.unicode.org/copyright.html
+#
+# GitHub Action configuration script for ICU continuous integration tasks.
+
+name: ICU Continuous Integration
```

Comment:
Since this name shows up for every single build line in the small status box, I wonder if maybe we could use a slightly shorter or abbreviated name, so that text doesn't overflow the space (?)

![image](https://user-images.githubusercontent.com/29107334/101952645-d7c00a00-3bad-11eb-8d33-799c4ed7f7e1.png)

For example, for the Azure CI builds I just used "CI" since you can see the blue azure logo. Maybe even "CI" would work for this too, since we can see the GitHub logo?

Reply:
As another alternative, we can call it "GH" or "GHA" to help distinguish it from the Azure CI.

Reply:
Good idea, "GH" or "GHA", both sgtm.